internal/proto/controller/api/services/v1/scope_service.proto

```diff
@@ -87,6 +87,54 @@ service ScopeService {
       summary: "Deletes a Scope."
     };
   }
+
+  // ListKeys lists all the keys found in the scope specified. If the scope
+  // is not found an error is returned. If the scope is empty, the global
+  // scope is used.
+  rpc ListKeys(ListKeysRequest) returns (ListKeysResponse) {
+    option (google.api.http) = {
+      get: "/v1/scopes/{id}:list-keys"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "List all keys in a Scope."
+    };
+  }
+
+  // RotateKeys rotates and optionally rewraps all the keys found in the
+  // scope specified. If the scope is not found an error is returned. If
+  // the scope is empty, the global scope is used.
+  rpc RotateKeys(RotateKeysRequest) returns (RotateKeysResponse) {
+    option (google.api.http) = {
+      post: "/v1/scopes:rotate-keys"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Rotate all keys in a Scope."
+    };
+  }
+
+  // ListKeyVersionDestructionJobs lists any pending key version destruction jobs in the scope.
+  rpc ListKeyVersionDestructionJobs(ListKeyVersionDestructionJobsRequest) returns (ListKeyVersionDestructionJobsResponse) {
+    option (google.api.http) = {
+      get: "/v1/scopes/{scope_id}:list-key-version-destruction-jobs"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Lists all pending key version destruction jobs in the scope."
+    };
+  }
+
+  // DestroyKeyVersion destroys the specified key version. If this requires re-encrypting
+  // existing data, it will start an asynchronous process to complete this operation
+  // before destroying the key. Use ListKeyVersionDestructionJobs to monitor pending destruction jobs.
```

Comment:
Do you mean the first part (`DestroyKeyVersion destroys the specified key version`)? I could definitely reword it a bit ("starts the destruction of"?). Otherwise I don't know what you mean.

Reply:
This is fine.